source/GitHubMarkdownAnchorId/Config.plist

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>Actions</key>
+	<array>
+		<dict>
+			<key>After</key>
+			<string>paste-result</string>
+			<key>Script Interpreter</key>
+			<string>/usr/local/bin/bash</string>
+			<key>Shell Script File</key>
+			<string>slugify.sh</string>
+			<key>Title</key>
+			<string>GitHub Anchor Header Link id</string>
```

Comment:
I'll rename the extension to somethign a bit snappier to publish it - I'm thinking "Slugify"